src/styles/blocks/header.scss

```
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  height: 60px;
+  /* I put height here because when I put it in .nav__link - it 
+  doesn`t affect form height. Also paddings doesn`t affect */
+  // height: 100%;
+
+  &__logo {
+    display: block;
+    width: 40px;
+    height: 40px;
+  }
+}
+  
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    list-style: none;
+    height: 100%;
+  }
+
+  &__item:not(:first-child) {
+    padding-left: 20px;
+  }
+  
+  &__link {
+    // position: relative;
+    font-style: normal;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $main_text_color;
+    padding: 23px 0;
+  }
```

Let's change it a bit
```suggestion
  &__link {
    position: relative;

    display: flex;
    align-items: center;

    height: 60px;
    
    font-style: normal;
    font-size: 12px;
    font-weight: 500;
    line-height: 14px;
    text-transform: uppercase;
    text-decoration: none;
    color: $main_text_color;
  }
```

---

Yes, I saw that bottom line is out of header, I asked about this problem in 1 message :)
Thank you, now it works as it should be